### PR TITLE
Fix bagel docker compose setup lakefs on startup

### LINKED ISSUE #4231: Docker compose (bagel) fail to setup on startup
Docker compose using local settings fail to setup and create repository on startup.
The lakefs-setup service used to setup lakefs while we used postgres, but using local settings the second instance can't access the local data found in lakefs service.

### PR DESCRIPTION
**Pull request by nopcoder**
- Fix docker compose bagel lakefs setup
- ignore failed setup and repo create

Fix #4231

### CODE REVIEW

File: deployments/compose/docker-compose.yml [hunk 1]
```diff
@@ -1,19 +1,33 @@
...
+      - "8000:8000"
     environment:
+      - LAKEFS_DATABASE_TYPE=local
+      - LAKEFS_BLOCKSTORE_TYPE=s3
+      - LAKEFS_BLOCKSTORE_S3_FORCE_PATH_STYLE=true
+      - LAKEFS_BLOCKSTORE_S3_ENDPOINT=http://minio:9000
+      - LAKEFS_BLOCKSTORE_S3_CREDENTIALS_ACCESS_KEY_ID=minioadmin
+      - LAKEFS_BLOCKSTORE_S3_CREDENTIALS_SECRET_ACCESS_KEY=minioadmin
       - LAKEFS_AUTH_ENCRYPT_SECRET_KEY=some random secret string
+      - LAKEFS_STATS_ENABLED=false
```
Comment (itaiad200) [inline deployments/compose/docker-compose.yml:19]: why false?

File: deployments/compose/docker-compose.yml [hunk 2]
```diff
@@ -1,19 +1,33 @@
...
       - LAKECTL_CREDENTIALS_ACCESS_KEY_ID=AKIAIOSFODNN7EXAMPLE
       - LAKECTL_CREDENTIALS_SECRET_ACCESS_KEY=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
-      - LAKECTL_SERVER_ENDPOINT_URL=http://lakefs:8000
-    entrypoint: ["wait-for", "lakefs:8000", "--", "sh", "-c",
-                 "lakefs setup --user-name docker --access-key-id AKIAIOSFODNN7EXAMPLE --secret-access-key wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY && lakectl repo create lakefs://example s3://example"]
+      - LAKECTL_SERVER_ENDPOINT_URL=http://localhost:8000
+    entrypoint: ["/bin/sh", "-c"]
+    command:
+        - |
+          lakefs setup --local-settings --user-name docker --access-key-id AKIAIOSFODNN7EXAMPLE --secret-access-key wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY || true
```
Comment (itaiad200) [inline deployments/compose/docker-compose.yml:27]: I wonder if we should have `lakectl setup` now, just like you can use it from the UI..

**Review by itaiad200 — APPROVED** [review]
See stats comment.

File: deployments/compose/docker-compose.yml (hunk 1 above)
Comment (nopcoder) [reply deployments/compose/docker-compose.yml:19]: my env - will revert

File: deployments/compose/docker-compose.yml (hunk 2 above)
Comment (nopcoder) [reply deployments/compose/docker-compose.yml:27]: You mean add lakectl command to perform setup?

Comment (nopcoder) [reply deployments/compose/docker-compose.yml:27]: Think we need to explain that when you use the 'local' option - all the lakefs command (database related) should be run on the same machine. Same place we enable that local is 'local' and you can't use more than one instance.

**Pull request merged**